shell/platform/tizen/external_texture_pixel_evas_gl.cc

```diff
@@ -59,14 +53,14 @@ bool ExternalTexturePixelGL::CopyPixelBuffer(size_t& width, size_t& height) {
   height = pixel_buffer->height;
 
   if (state_->gl_texture == 0) {
-    glGenTextures(1, &state_->gl_texture);
-    glBindTexture(GL_TEXTURE_2D, state_->gl_texture);
+    glGenTextures(1, (GLuint*)&state_->gl_texture);
```

Comment:
Could you please use c++ style cast instead of c-style cast?

Reply:
done